statefun-examples/statefun-ridesharing-example/statefun-ridesharing-example-simulator/pom.xml

```diff
@@ -169,6 +178,24 @@ under the License.
                                 <exclude>META-INF/*.RSA</exclude>
                             </excludes>
                         </filter>
+                        <filter>
+                            <artifact>org.apache.kafka:*</artifact>
+                            <excludes>
+                                <exclude>kafka/kafka-version.properties</exclude>
+                                <exclude>LICENSE</exclude>
+                                <!-- Does not contain anything relevant.
+                                    Cites a binary dependency on jersey, but this is neither reflected in the
+                                    dependency graph, nor are any jersey files bundled. -->
+                                <exclude>NOTICE</exclude>
```

Comment:
Are we including the contents of Kafka's `NOTICE` file somewhere?

We have to forward the Kafka NOTICE file, even if we disagree with it's contents (at least that's how I understand the ASL 2.0)

Reply:
To address that we can manually merge it to our NOTICE file (its not getting picked up by the `ApacheNoticeResourceTransformer`), but as far as I can tell Flink itself is also excluding the contents completely.

Reply:
How it is being done in Flink may be wrong though -
I won't be against merging the contents to our NOTICE file, to be safe.

Reply:
We have to only forward parts of the NOTICE relevant to the parts we redistribute, according so ASLv2.

Reply:
Okay, thanks.
Then this review item is resolved.